macos/FluentUI/Separator.swift

```diff
@@ -0,0 +1,82 @@
+//
+//  Copyright (c) Microsoft Corporation. All rights reserved.
+//  Licensed under the MIT License.
+//
+
+import AppKit
+
+@objc(MSFSeparatorStyle)
+public enum SeparatorStyle: Int {
+    case `default`
+    case shadow
+
+    fileprivate var color: NSColor {
+		switch self {
+		case .default:
+			if #available(OSX 10.14, *) {
+				return NSColor.separatorColor
+
+			} else {
+				return NSColor.systemGray
+			}
+		case .shadow:
+			return NSColor.gridColor
+        }
+    }
+}
+
+@objc(MSFSeparatorOrientation)
+public enum SeparatorOrientation: Int {
+    case horizontal
+    case vertical
+}
+
+
+@objc(MSFSeparator)
+open class Separator: NSView {
+	private var orientation: SeparatorOrientation = .horizontal
```

Comment:
>private [](start = 1, length = 7)

in Fluent iOS we place public methods before private methods. I like that guideline for readability, what do you think?

Reply:
Here's the relevant Microsoft swift style guide page: https://microsoft.github.io/swift-guide/FileContentOrdering.html

Reply:
Was just about to link that page :D 
Thanks, I wrote this before I was aware of that section. I'll update to reflect